pkg/api/opssight/v1/types.go

```diff
@@ -107,11 +108,15 @@ type PodPerceiver struct {
 
 // Perceiver stores the Perceiver configuration
 type Perceiver struct {
-	EnableImagePerceiver      bool          `json:"enableImagePerceiver"`
-	EnablePodPerceiver        bool          `json:"enablePodPerceiver"`
-	PodPerceiver              *PodPerceiver `json:"podPerceiver,omitempty"`
-	AnnotationIntervalSeconds int           `json:"annotationIntervalSeconds"`
-	DumpIntervalMinutes       int           `json:"dumpIntervalMinutes"`
+	EnableImagePerceiver             bool          `json:"enableImagePerceiver"`
+	EnableArtifactoryPerceiver       bool          `json:"enableArtifactoryPerceiver"`
+	EnableArtifactoryPerceiverDumper bool          `json:"enableArtifactoryPerceiverDumper"`
+	EnableQuayPerceiver              bool          `json:"enableQuayPerceiver"`
+	EnablePodPerceiver               bool          `json:"enablePodPerceiver"`
+	PodPerceiver                     *PodPerceiver `json:"podPerceiver,omitempty"`
+	AnnotationIntervalSeconds        int           `json:"annotationIntervalSeconds"`
+	DumpIntervalMinutes              int           `json:"dumpIntervalMinutes"`
+	Expose                           string        `json:"expose"`
```

Comment:
what are we exposing for Perceivers?

Reply:
For Webhooks, they send a JSON to the perceivers with Image Info which needs to be scanned